envelope.md

```diff
@@ -55,6 +52,28 @@ envelopes with individual signatures.
 }
 ```
 
+### No signatures
+
+An envelope MAY have zero signatures to indicate unsigned data. For example,
+[SLSA 1](https://github.com/slsa-framework/slsa) does not require authenticated
+[in-toto provenance](https://github.com/in-toto/attestation/blob/main/spec/predicates/provenance.md),
+in which case an envelope with zero signatures may be useful.
+
+```json
+{
+  "payload": "<Base64(SERIALIZED_BODY)>",
+  "payloadType": "<PAYLOAD_TYPE>"
+}
+```
+
+### Parsing rules
+
+*   The following fields are REQUIRED and MUST be set, even if empty: `payload`,
+    `payloadType`, `signature.sig`.
+*   The following fields are OPTIONAL and MAY be unset: `signature`,
+    `signature.keyid`. An unset field MUST be treated the same as set-but-empty.
+*   Consumers MUST ignore unrecognized fields.
```

Comment:
```suggestion
*   There MAY be additional fields added, but consumers MUST ignore unrecognized fields.
```

Reply:
I think it'd be nice to make this explicit.

Reply:
Is the intention to say that **producers** MAY add additional fields? I'd like to add a subject to make it more clear.

Reply:
Yes, that would probably make more sense.

Reply:
Fixed.